test/relayHub/RelayHub.test.ts

```diff
@@ -1,1177 +1,507 @@
-
-//         const relayOwnerBalanceBefore = toBN(
-//           await web3.eth.getBalance(relayOwner)
-//         );
-
-//         const gasPrice = toBN('60000000');
-//         const txResponse = await mockRelayHub.withdrawStake(relayManager, {
-//           from: relayOwner,
-//           gasPrice,
-//         });
-
-//         //Getting the gas used in order to calculate the original balance of the account
-//         const rbtcUsed = toBN(
-//           (await web3.eth.getTransactionReceipt(txResponse.tx))
-//             .cumulativeGasUsed
-//         ).mul(gasPrice);
-
-//         const relayOwnerBalanceAfter = toBN(
-//           await web3.eth.getBalance(relayOwner)
-//         );
-//         assert.isTrue(
-//           relayOwnerBalanceAfter.eq(
-//             relayOwnerBalanceBefore.sub(rbtcUsed).add(stakeBalanceBefore)
-//           ),
-//           'Withdraw/unstake process have failed'
-//         );
-
-//         stakeInfo = await mockRelayHub.getStakeInfo(relayManager);
-//         const stakeAfterWithdraw = toBN(stakeInfo.stake);
-
-//         //Verifying there are no more stake balance for the manager
-//         assert.isTrue(stakeAfterWithdraw.isZero(), 'Stake must be zero');
-
-//         relayWorkersBefore = await mockRelayHub.workerCount(relayManager);
-//         assert.equal(
-//           relayWorkersBefore.toNumber(),
-//           1,
-//           `Initial workers must be zero but was ${relayWorkersBefore.toNumber()}`
-//         );
-
-//         const calculatedAddr = await factory.getSmartWalletAddress(
-//           gaslessAccount.address,
-//           constants.ZERO_ADDRESS,
-//           relayRequestMisbehavingVerifier.request.index
-//         );
-//         await token.mint('1', calculatedAddr);
-
-//         await assert.isRejected(
-//           mockRelayHub.deployCall(
-//             relayRequestMisbehavingVerifier,
-//             signatureWithMisbehavingVerifier,
-//             { from: relayWorker, gas, gasPrice }
-//           ),
-//           ERR_UNSTAKED,
-//           'Deploy was processed successfully'
-//         );
-//       });
-
-//       it('Should fail when registering with no workers assigned to the Manager', async () => {
-//         await mockRelayHub.stakeForAddress(relayManager, 1000, {
-//           value: oneRBTC,
-//           from: relayOwner,
-//         });
-
-//         const relayWorkersBefore = await mockRelayHub.workerCount(
-//           relayManager
-//         );
-//         assert.equal(
-//           relayWorkersBefore.toNumber(),
-//           0,
-//           `Initial workers must be zero but was ${relayWorkersBefore.toNumber()}`
-//         );
-
-//         await assert.isRejected(
-//           mockRelayHub.registerRelayServer(url, {
-//             from: relayManager,
-//           }),
-//           'no relay workers',
-//           'Relay Server was successfully registered'
-//         );
-//       });
-//     });
-//   }
-// );
+  }
+);
+
```

Comment:
Please use lower case for the name of this files.

Reply:
Done